tests/end-to-end/director/labeling_api_test.go

```diff
@@ -0,0 +1,845 @@
+package director
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	gcli "github.com/machinebox/graphql"
+)
+
+func TestCreateLabelWithoutLabelDefinition(t *testing.T) {
+	//Create application
```

Comment:
I think this whole Create application block could be moved to helper function because it's used very often in tests, consider moving repetitive blocks like that to helper functions / fixtures. (be careful with defers though)

Reply:
Exactly. Please try to refactor these tests.

Reply:
moved common parts to fixtures_queries_test and fixtures_request_test files